src/FSLibrary/MissionMixedImageLoadGeneration.fs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+module MissionMixedImageLoadGeneration
+
+open stellar_dotnet_sdk
+open StellarCoreSet
+open StellarMissionContext
+open StellarFormation
+open StellarStatefulSets
+open StellarSupercluster
+open StellarCoreHTTP
+
+let mixedImageLoadGeneration (context: MissionContext) =
+    let oldNodeCount = context.oldImageNodeCount
+    let newNodeCount = 3 - context.oldImageNodeCount
+    let oldKeys = Array.init oldNodeCount (fun _ -> KeyPair.Random())
+    let newKeys = Array.init newNodeCount (fun _ -> KeyPair.Random())
+
+    let newImage = context.image
+    let oldImage = GetOrDefault context.oldImage newImage
+
+    let oldName = "core-old"
+    let newName = "core-new"
+
+    // Allow 2/3 nodes consensus, so that one image version could fork the network
+    // in case of bugs.
+    let qSet =
+        CoreSetQuorumListWithThreshold(([ CoreSetName oldName; CoreSetName newName ], 51))
```

Comment:
minor: would a setup of 4 nodes with default threshold achieve the same result? Then we wouldn't need the additional threshold functionality.

Reply:
I guess the question is whether probability of 3 nodes simultaneously arriving at the same value (e.g. `txSetHash` for our case) is significantly different from probability of 2 nodes doing the same. I don't really know the answer, do you have any idea? Also, my other motivation was to have the same setup as testnet.

Reply:
Yeah, it's true that it's probably better to replicate the testnet setup (for possible future issues as well). I think we can just keep your change as-is then: my original comment was around whether we needed the new code, but it's not much code to begin with, so not a big deal.